tools/Microsoft.AspNet.Server.Kestrel.LibuvCopier/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.AspNet.Server.Kestrel.LibuvCopier
+{
+    public class Program
+    {
+        public void Main(string[] args)
+        {
+            try
+            {
+                var lockJson = JObject.Parse(File.ReadAllText("project.lock.json"));
+                var dnxHome = Environment.GetEnvironmentVariable("DNX_HOME");
```

Comment:
Need to add fallback if DNX_HOME is not defined - this is what causing Travis/Appveyor failures.